modules/builds-strategy-docker-entitled-satellite.adoc

```diff
@@ -18,6 +18,13 @@ Use the following as an example Dockerfile to install content with Satellite:
 [source,terminal]
 ----
 FROM registry.redhat.io/ubi9/ubi:latest
-RUN dnf search kernel-devel --showduplicates && \
-        dnf install -y kernel-devel
+RUN rm -rf /etc/rhsm-host <1>
+RUN yum --enablerepo=codeready-builder-for-rhel-9-x86_64-rpms install \ <2>
+    nss_wrapper \
+    uid_wrapper -y && \
+    yum clean all -y
+RUN ln -s /run/secrets/rhsm /etc/rhsm-host <3>
 ----
+<1> You must include the command to remove the `/etc/rhsm-host` directory and all its contents in your Dockerfile before executing any `yum` or `dnf` commands.
```

Comment:
```suggestion
<1> You must include the command to remove the `/etc/rhsm-host` directory and its content in your Dockerfile before executing any `yum` or `dnf` commands.
```

Reply:
Sorry for the confusion. Instead of mentioning it in a callout, I think in the beginning it would be better to clarify that the below configuration is of a Dockerfile.